components/mdns/.cz.yaml

```diff
@@ -3,6 +3,6 @@ commitizen:
   bump_message: 'bump(mdns): $current_version -> $new_version'
   pre_bump_hooks: python ../../ci/changelog.py mdns
   tag_format: mdns-v$version
-  version: 1.7.0
+  version: 1.7.1
```

Comment:
Is the bumped version correct? We are adding anew feature so shouldn't it be 1.8.0?

Reply:
yes, you're right, it should be `1.8.0` -> fixed